generic-contracts/src/main/java/com/scalar/dl/genericcontracts/table/v1_0_0/Create.java

```diff
@@ -0,0 +1,82 @@
+package com.scalar.dl.genericcontracts.table.v1_0_0;
```

Comment:
Since users need to register new updated (or bug-fixed) versions of these generic contracts with different binary names, e.g., `com.scalar.dl.genericcontracts.table.v1_0_0.Create` and `com.scalar.dl.genericcontracts.table.v1_0_1.Create` in ScalarDL, we introduce package-based semantic versioning, although it’s weird for common practice in Java.